frontend/src/app/_components/creator-list/creator-list.component.html

```diff
@@ -1,5 +1,8 @@
 <div class="creator-list row">
-    <div class="col-md-4 col-12 col-sm-6 col-lg-3" *ngFor="let creator of creators">
+    <div *ngIf="creators.length === 0">
+        <p>Nothing to see here.</p>
```

Comment:
Is there a situation where we'll have 0 creators? We should probably explain what it means instead of saying "Nothing to see here." Or if this is just temporary, add a TODO so we're more likely to fix it later?

Reply:
only if something is very broken.